cdm/src/test/java/ucar/nc2/ft2/coverage/simpgeometry/TestGetGeometryType.java

```diff
@@ -0,0 +1,44 @@
+package ucar.nc2.ft2.coverage.simpgeometry;
+
+import java.io.IOException;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import ucar.nc2.dataset.NetcdfDataset;
+import ucar.unidata.util.test.TestDir;
+
+/**
+ * Tests ability to get Simple Geometry Type using Simple Geometry Reader.
+ * 
+ * @author wchen@usgs.gov
+ *
+ */
+public class TestGetGeometryType {
+	
+	private SimpleGeometryReader openReaderOverFile(String filename) throws IOException {

```

Comment:
Good call to deduplicate via this utility method!

Reply:
Thanks! Yeah it makes it very succinct.